corehq/apps/app_manager/models.py

```diff
@@ -3340,6 +3341,53 @@ def get_filter_value(self, user):
         return DateSpan(startdate=start_date, enddate=end_date)
 
 
+class CustomDatespanFilter(ReportAppFilter):
+    operator = StringProperty(
+        choices=[
+            '=',
+            '<=',
+            '>=',
+            '>',
+            '<',
+            'between'
+        ],
+        required=True,
+    )
+    date_number = StringProperty(required=True)
+    date_number2 = StringProperty()
```

Comment:
eh, nm, forgot that `date_number2` isn't necessarily the later date, maybe there isn't a much better name for it